minikube-maven-plugin/README.md

```diff
@@ -4,7 +4,7 @@ Minikube Maven Plugin
 ======================
 This plugin provides tasks and configuration to manage a minikube lifecycle for Maven developers.
 
-Now available on [Maven Central]().
+Now available on [Maven Central](https://mvnrepository.com/artifact/com.google.cloud.tools/minikube-maven-plugin).
```

Comment:
This is not maven central.

Reply:
Changed the wording to just say `the Central repository`.